src/test/features/loading-spec.ts

```diff
@@ -178,6 +178,34 @@ suite('ModelViewerElementBase with LoadingMixin', () => {
         });
       });
 
+      suite('when element starts in non-rendered state', () => {
+        setup(() => {
+          element.style.display = 'none';
+        });
+
+        suite('model-visibility event', () => {
+          test('is dispatched', async () => {
+            const modelBecomesVisible =
+                waitForEvent(element, 'model-visibility');
+            element.src = ASTRONAUT_GLB_PATH;
+            await modelBecomesVisible;
+          });
+        });
+
+        suite('modelIsVisible', () => {
+          test(
+              'is true when the element eventually becomes renderable',
+              async () => {
+                const modelBecomesVisible =
+                    waitForEvent(element, 'model-visibility');
+                element.src = ASTRONAUT_GLB_PATH;
+                await modelBecomesVisible;
```

Comment:
Doesn't this test include the previous one?

Reply:
It depends on the same circumstances as the previous one. I tend to err on the side of overlap if it seems like there are two distinct behaviors that should be ensured. For example, I wouldn't remove a test that checks for the `'load'` event just because 50 other tests depend on observing that event as part of their setup.